rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -462,19 +463,41 @@ public void setPendingFederation(PendingFederation federation) {
      */
     public void savePendingFederation() {
         if (shouldSavePendingFederation) {
-            RepositorySerializer<PendingFederation> serializer = BridgeSerializationUtils::serializePendingFederationOnlyBtcKeys;
-
-            if (activations.isActive(RSKIP123)) {
-                // we only need to save the standard part of the fed since the emergency part is constant
-                saveStorageVersion(
-                    PENDING_FEDERATION_FORMAT_VERSION.getKey(),
-                    STANDARD_MULTISIG_FEDERATION.getFormatVersion()
-                );
-                serializer = BridgeSerializationUtils::serializePendingFederation;
+            if (!activations.isActive(RSKIP123)) {
+                savePendingFederationOnlyBtcKeys(pendingFederation);
+            }
+            else {
+                savePendingFederation(pendingFederation);
             }
+        }
+    }
+
+    /**
+     * Save the pending federation before RSKIP123
+     */
+    public void savePendingFederationOnlyBtcKeys(PendingFederation pendingFederation) {
+        byte[] fedSerialized = null;
+        if (pendingFederation != null) {
```

Comment:
So what happens if the pendingFederation is null? You store `null` in the storage? Is that the expected behaviour?

Does it make sense that pendingFederation is null if `shouldSavePendingFederation` is true?